tests/e2e/carInfo/carInfo.spec.js

```diff
@@ -0,0 +1,7 @@
+describe('Car Info', function(){
+  it('should go to carInfo page', function() {
+    browser.get(browser.baseUrl + 'carInfo');
+    lement(by.css(".arrow")).click();
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carInfo");
```

Comment:
Use some other jasmine matcher instead so it does not relies on the environment you're running tests against.
for example:
`expect(browser.getCurrentUrl()).toContain("/#/carInfo");`